sdk/core/core/src/az_http_policy.c

```diff
@@ -6,6 +6,7 @@
 #include <az_credentials.h>
 #include <az_http.h>
 #include <az_http_internal.h>
+#include <az_http_transport.h>
```

Comment:
az_http_policy.h already includes az_http_transport.h (for _az_http_request at least)

Reply:
removed